plan/2.0 Expand Training.md

```diff
@@ -38,338 +16,519 @@
+- **M4**: Creation of materials for other channels
 
-### Time & resource estimate
+  - [ ] Deliver twelve "thought leadership" secure development blogs
+  - [ ] Deliver 2 secure development OSSF webinars
 
-- Y1 - XX hours to modify existing class materials to become training camp-ready
-- Y1 - XX hours training the trainers for camps
-- Y1 - 24 OSSF and volunteer hours coordinating scheduling
+#### Time & Resource Estimate
 
-## 2.7 Goal: Develop in-person educator-led variations of training materials and devlier bootcamps at conferences [Stream 1?]
+| Milestone | SKF Ops | Contractor(s) | Time Estimate |  Year  |
+| :-------: | :-----: | :-----------: | :-----------: | :----: |
+|  **M1**   |   ✅    |       -       |     10hrs     | Year 1 |
+|  **M2**   |   ⛔    |       1       |     40hrs     | Year 1 |
+|  **M3**   |   🤝    |       -       |    180hrs     | Year 1 |
+|  **M4**   |   ⛔    |       1       |    160hrs     | Year 1 |
 
-### Explanation: Modify existing materials and develop a training course that could be delivered at conferences including hands-on lab work
+> **CONTRACTOR - TOTAL**: 380 hours
+> **SKF OPS TEAM - TOTAL**: 80 hours
 
-### Key Steps/Milestones
+Y1 - Hire Education Content Developer (1 person x $ 300K): $ 300K
 
-- **M1**: Create job descriptions for content creators
-- **M2**: hire content creators
-- **M3**: Review existing content
-- **M4**: Generate modified class
-- **M5**: Schedule 2 bootcamps at LF-sponsored conferences
-- **M6**: train trainers
-- YEAR 2
-- **M7**: adjust bootcamp for new content delivered during YEAR 1
+---
 
-### Time & resource estimate
+### 2.7.1: Educational content development and management of blogs, webinars, podcasts
 
-- Y1 - XX hours to modify existing class materials to become conference-ready bootcamp track
-- Y1 - XX hours training the trainers for bootcamp
-- Y1 - 24 OSSF and volunteer hours coordinating conference scheduling
+### Explanation: We want to create different delivery methods for the materials we have to also reach other types of persona's using blogs, podcasts and webinars
 
-### Time & resource estimate
+#### Key Steps/Milestones
 
-4 person x $300K): $1.2M
+- **M1**: Identification of the different channels of the delivery of materials
 
-## 2.8 Goal: Partnerships with traditional educational bodies around the globe
+  - [ ] Identify and select channels content will be delivered in
+  - [ ] Create a roadmap for content creation and publication
 
-### Explanation: Develop 5-10 meaningful relationships with leading universities to add secure development coursework to university curricula as requirement for undergraduate courses
+- **M2**: Create content for channels
 
-### Key Steps/Milestones
+  - [ ] Identify and select channels content will be delivered in
+  - [ ] Develop in-person educator-led variations of training materials (e.g., with hands-on lab work) & go to conferences to encourage training
```

Comment:
This second milestone looks like it should be moved up to where the focus on conferences is. Unless the conference itself constitutes a content channel, then that should be moved down here and consolidated. which would make this milestone read more specific: Develop channel appropriate content variations of training materials i.e. tutorials for in-person conferences, etc.